components/Board/Profile.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+export default function Profile(props) {
+	const mailTo = "mailTo:" + props.person.Email;
+	return (
+		<div className='profileCard flex w-3/6'>
+			<div className='max-w-sm rounded-md shadow-md dark:dark:bg-gray-900 dark:dark:text-gray-100'>
+				<h1 className='text-center text-3xl font-semibold tracking-wide'>
+					{props.person.Position}
```

Comment:
variable names should be lowercase. There is no error causing rule, instead its for practice reasons since `Position` would imply a component. If the variable is multiple words, then camel casing (multipleWordVariable) should be used. Having the structure lets you quickly assume things and go through code more easily. Make sure to add these changes to the following variables below.